internal/buildscripts/packaging/installer/install.sh

```diff
@@ -910,48 +949,54 @@ Auto Instrumentation:
+                                        appended to the OTEL_RESOURCE_ATTRIBUTES environment variable.
                                         (default: empty)
-  --service-name <name>                 Override the auto-generated service names for all instrumented Java applications
-                                        on this host with '<name>'.
-                                        Only applicable if the '--with-instrumentation' or
-                                        '--with-systemd-instrumentation' option is also specified.
+  --service-name <name>                 Override the auto-generated service names for all instrumented applications
+                                        on this host with the specified value. The value will be set to the
+                                        OTEL_SERVICE_NAME environment variable.
                                         (default: empty)
-  --otlp-endpoint <host:port>           Set the OTLP gRPC endpoint for captured traces.
-                                        Only applicable if the '--with-systemd-instrumentation' option is also specified.
-                                        (default: http://LISTEN_INTERFACE:4317 where LISTEN_INTERFACE is the value from
-                                        the --listen-interface option if specified, or "127.0.0.1" otherwise)
-  --[enable|disable]-profiler           Enable or disable AlwaysOn Profiling.
-                                        Only applicable if the '--with-instrumentation' or
-                                        '--with-systemd-instrumentation' option is also specified.
+  --otlp-endpoint <host:port>           Set the OTLP endpoint for captured traces, logs, and metrics for all activated
+                                        SDKs. The value will be set to the OTEL_EXPORTER_OTLP_ENDPOINT environment
+                                        variable.
+                                        (default: empty, i.e. defer to the default OTEL_EXPORTER_OTLP_ENDPOINT value for
+                                        each activated SDK)
+  --otlp-endpoint-protocol <protocol>   Set the protocol for the configured OTLP endpoint, for example "grpc" or
+                                        "http/protobuf". The value will be set to the OTEL_EXPORTER_OTLP_PROTOCOL
+                                        environment variable.
+                                        Only applicable if the "--otlp-endpoint <host:port>" option is also specified.
+                                        (default: empty, i.e. defer to the default OTEL_EXPORTER_OTLP_PROTOCOL value for
+                                        each activated SDK)
+  --metrics-exporter <exporters>        Comma-separated list of exporters for collected metrics by all activated SDKs,
+                                        for example "otlp,zipkin". Set the value to "none" to disable collection and
```

Comment:
small nit: zipkin would be traces only 
```suggestion
                                        for example "otlp,prometheus". Set the value to "none" to disable collection and
```